<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QDockWidget" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>440</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"> QSplitter::handle:horizontal {
     width: 3px;
 }

 QSplitter::handle:vertical {
     height: 3px;
 }

QToolButton:hover {
	border-radius: 5px;
	border: none;
	padding: 2px;
	/*color: darkgray;*/
	background-color: rgb(232, 232, 232);
}

QToolButton:hover {
    background-color: rgb(128, 128, 128);
    color: darkgray;
}</string>
  </property>
  <property name="windowTitle">
   <string notr="true">Notepad</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="searchEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 7px;
padding: 0 8px;
/*
background-color: rgb(240, 240, 240);
color: rgb(48, 48, 48);
*/</string>
          </property>
          <property name="inputMask">
           <string notr="true"/>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Search</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="undoButton">
          <property name="styleSheet">
           <string notr="true">border-radius: 5px;
border: none;
padding: 2px;
/*color: darkgray;*/
background-color: rgb(232, 232, 232);</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/img/icons/undo.svg</normaloff>:/img/icons/undo.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="redoButton">
          <property name="styleSheet">
           <string notr="true">border-radius: 5px;
border: none;
padding: 2px;
/*color: darkgray;*/
background-color: rgb(232, 232, 232);

</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/img/icons/redo.svg</normaloff>:/img/icons/redo.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #333;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QToolButton" name="boldButton">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="toolTip">
              <string notr="true">Bold</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton { /* all types of tool button */
     border: 5px solid #333;
     border-left: 10px solid #333;
	 border-right: 10px solid #333;
     border-radius: 0px;
     background-color: #333;
	 color: rgb(172, 174, 175);
 }

QToolButton:hover {
     border: 5px solid #444;
     border-radius: 0px;
     background-color: #444;
}

QToolButton:pressed {
     border: 5px solid #2180a9;
     border-radius: 0px;
     background-color: #2180a9;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
}</string>
             </property>
             <property name="text">
              <string notr="true">B</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="italicsButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="toolTip">
              <string notr="true">Italics</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton { /* all types of tool button */
     border: 5px solid #333;
     border-left: 10px solid #333;
	 border-right: 10px solid #333;
     border-radius: 0px;
     background-color: #333;
	 color: rgb(172, 174, 175);
 }

QToolButton:hover {
     border: 5px solid #444;
     border-radius: 0px;
     background-color: #444;
 }

QToolButton:pressed {
     border: 5px solid #2180a9;
     border-radius: 0px;
     background-color: #2180a9;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
}</string>
             </property>
             <property name="text">
              <string notr="true">I</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="underlineButton">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <underline>true</underline>
              </font>
             </property>
             <property name="toolTip">
              <string notr="true">Underline</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton { /* all types of tool button */
     border: 5px solid #333;
     border-left: 10px solid #333;
	 border-right: 10px solid #333;
     border-radius: 0px;
     background-color: #333;
	 color: rgb(172, 174, 175);
 }

QToolButton:hover {
     border: 5px solid #444;
     border-radius: 0px;
     background-color: #444;
 }

QToolButton:pressed {
     border: 5px solid #2180a9;
     border-radius: 0px;
     background-color: #2180a9;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
}</string>
             </property>
             <property name="text">
              <string notr="true">U</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="h1Button">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string notr="true">Title 1</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton { /* all types of tool button */
     border: 5px solid #333;
     border-left: 10px solid #333;
	 border-right: 10px solid #333;
     border-radius: 0px;
     background-color: #333;
	 color: rgb(172, 174, 175);
 }

QToolButton:hover {
     border: 5px solid #444;
     border-radius: 0px;
     background-color: #444;
 }

QToolButton:pressed {
     border: 5px solid #2180a9;
     border-radius: 0px;
     background-color: #2180a9;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
}</string>
             </property>
             <property name="text">
              <string>H1</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="h2Button">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string notr="true">Title 2</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton { /* all types of tool button */
     border: 5px solid #333;
     border-left: 10px solid #333;
	 border-right: 10px solid #333;
     border-radius: 0px;
     background-color: #333;
	 color: rgb(172, 174, 175);
 }

QToolButton:hover {
     border: 5px solid #444;
     border-radius: 0px;
     background-color: #444;
 }

QToolButton:pressed {
     border: 5px solid #2180a9;
     border-radius: 0px;
     background-color: #2180a9;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
}</string>
             </property>
             <property name="text">
              <string>H2</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="h3Button">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string notr="true">Title 3</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton { /* all types of tool button */
     border: 5px solid #333;
     border-left: 10px solid #333;
	 border-right: 10px solid #333;
     border-radius: 0px;
     background-color: #333;
	 color: rgb(172, 174, 175);
 }

QToolButton:hover {
     border: 5px solid #444;
     border-radius: 0px;
     background-color: #444;
 }

QToolButton:pressed {
     border: 5px solid #2180a9;
     border-radius: 0px;
     background-color: #2180a9;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
}</string>
             </property>
             <property name="text">
              <string>H3</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="fontButton">
             <property name="font">
              <font>
               <family>Big Caslon</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string notr="true">Change font</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton { /* all types of tool button */
     border: 5px solid #333;
     border-left: 10px solid #333;
	 border-right: 10px solid #333;
     border-radius: 0px;
     background-color: #333;
	 color: rgb(172, 174, 175);
 }

QToolButton:hover {
     border: 5px solid #444;
     border-radius: 0px;
     background-color: #444;
 }

QToolButton:pressed {
     border: 5px solid #2180a9;
     border-radius: 0px;
     background-color: #2180a9;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
}</string>
             </property>
             <property name="text">
              <string>A</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QSplitter::handle {
    height: 2px;
	 background-color: rgb(255, 255, 255);
    image: url(:/img/icons/tabs.svg);
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPlainTextEdit" name="notepadTextEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Monaco</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="lineWrapMode">
            <enum>QPlainTextEdit::NoWrap</enum>
           </property>
           <property name="plainText">
            <string/>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="previewTextEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Anonymous Pro</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="lineWrapMode">
            <enum>QPlainTextEdit::NoWrap</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionDisassmble_bytes">
   <property name="text">
    <string>Disassmble bytes at </string>
   </property>
   <property name="toolTip">
    <string>Disassmble bytes at </string>
   </property>
  </action>
  <action name="actionDisassmble_function">
   <property name="text">
    <string>Disassmble function at </string>
   </property>
   <property name="toolTip">
    <string>Disassmble function at </string>
   </property>
  </action>
  <action name="actionHexdump_bytes">
   <property name="text">
    <string>Hexdump bytes at </string>
   </property>
   <property name="toolTip">
    <string>Hexdump bytes at </string>
   </property>
  </action>
  <action name="actionCompact_Hexdump">
   <property name="text">
    <string>Compact Hexdump at </string>
   </property>
   <property name="toolTip">
    <string>Compact Hexdump at </string>
   </property>
  </action>
  <action name="actionHexdump_function">
   <property name="text">
    <string>Hexdump function at</string>
   </property>
   <property name="toolTip">
    <string>Hexdump function at</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
